ACID/acid_classic.sql

```diff
@@ -9197,8 +9197,7 @@ INSERT INTO `creature_ai_scripts` (`id`,`creature_id`,`event_type`,`event_invers
 ('319801','3198','1','0','100','0','1000','1000','0','0','0','0','56','0','0','0','0','0','0','0','0','0','0','0','Burning Blade Apprentice - Despawn Guardians on OOC'),
 ('319802','3198','1','0','100','0','2000','2000','0','0','0','0','11','12746','0','0','0','0','0','0','0','0','0','0','Burning Blade Apprentice - Cast Summon Voidwalker on OOC'),
 ('319803','3198','4','0','100','0','0','0','0','0','0','0','57','2','35','0','0','0','0','0','0','0','0','0','Burning Blade Apprentice - Enable Range Mode on Aggro'),
-('319804','3198','2','0','100','1024','15','0','0','0','0','0','25','0','0','0','1','1150','0','0','0','0','0','0','Burning Blade Apprentice - Flee at 15% HP'),
-('319805','3198','9','0','100','1025','0','40','3800','5200','0','0','11','20791','1','256','0','0','0','0','0','0','0','0','Burning Blade Apprentice - Cast Shadow Bolt'),
+('319804','3198','9','0','100','1025','0','40','3800','5200','0','0','11','20791','1','256','0','0','0','0','0','0','0','0','Burning Blade Apprentice - Cast Shadow Bolt'),
```

Comment:
Also add something like this:
```
('319804','3198','0','0','100','1281','0','0','3800','5200','0','0','11','20791','1','256','0','0','0','0','0','0','0','0','Burning Blade Apprentice - Cast Fireball (Range Mode)'),
('319805','3198','0','0','100','1537','0','0','5000','20000','0','0','11','20791','1','256','0','0','0','0','0','0','0','0','Burning Blade Apprentice - Cast Fireball (Melee Mode)'),
```
to decrease their frequency of casting when being fought in melee combat.

Reply:
Done, thanks!